test/parallel_api/numeric/numeric.ops/exclusive_scan.pass.cpp

```diff
@@ -0,0 +1,102 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include _PSTL_TEST_HEADER(execution)
+#include _PSTL_TEST_HEADER(algorithm)
+#include _PSTL_TEST_HEADER(numeric)
+
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+
+#include "support/sycl_alloc_utils.h"
+
+template <sycl::usm::alloc alloc_type>
+void
+test_with_usm(sycl::queue& q, const ::std::size_t __count)
```

Comment:
There is non consistency in the file  - uglified identificators or non-uglified.
Actually, we follow "uglification" just for the implementation, not for the test.

Reply:
Fixed